README.md

```diff
@@ -11,33 +11,155 @@ The controller reconciles `ModelRegistry` Custom Resources to create a service f
 You’ll need a Kubernetes cluster to run against. You can use [KIND](https://sigs.k8s.io/kind) to get a local cluster for testing, or run against a remote cluster.
 **Note:** Your controller will automatically use the current context in your kubeconfig file (i.e. whatever cluster `kubectl cluster-info` shows).
 
-`ModelRegistry` service needs a PostgreSQL or MySQL database. A sample Postgres configuration for testing is included 
+`ModelRegistry` service needs a PostgreSQL or MySQL database. A sample Postgres configuration for testing (without TLS security) is included 
```

Comment:
There is some ambiguity in the way TLS is used in the README. All the DB samples are configured without TLS DB connections, since it requires creating certs, secrets, etc. to inject them into the model registry CR. 
@HumairAK had to spend a few days to get a MySQL connection with TLS working with rhoai mlmd image. 
So, we don't have ootb samples for TLS DB connections. The TLS samples refer to using TLS secured Istio Gateway endpoints. I'll work on re-wording the README to clarify this. 
We can create another jira issue to add TLS DB connection samples (setup would have to be elaborate to explain cert creation, secret creation, etc.) in another PR. 
wdyt?

Reply:
Yes feel free to do it in another PR, I agree that it is a bit off topic here 👍